### ISSUE TITLE
Machine Learning Engineer

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#7031]
Machine Learning Engineer<br>
http://bit.ly/2hDyt8p<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0001/8122/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> San Francisco, CA<br>
<strong>URL:</strong> <a href="https://engineering.doximity.com/">https://engineering.doximity.com/</a></p>
<div><span>Why work at Doximity?</span></div>
<div><span><br></span></div>
<div><span>Doximity is the leading social network for healthcare professionals with over 70% of U.S. doctors as members. We have strong revenues, real market traction, and we're putting a dent in the inefficiencies of our $2.5 trillion U.S. healthcare system. After the iPhone, Doximity is the fastest adopted product by doctors of all time. Our founder, Jeff Tangney, is the founder &amp; former President and COO of Epocrates (IPO in 2010), and Nate Gross is the founder of digital health accelerator RockHealth. Our investors include top venture capital firms who've invested in Box, Salesforce, Skype, SpaceX, Tesla Motors, Twitter, Tumblr, Mulesoft, and Yammer. Our beautiful offices are located in SoMa San Francisco.</span></div>
<div><br></div>
<div>
<span>Skills &amp; Requirements:</span><br>
</div>
<div><span><br></span></div>
<div>
<span>-3+ years of industry experience; M.S. in Computer Science or other relevant technical field preferred.</span><br>
</div>
<div>
<span>-3+ years experience collaborating with data science and data engineering teams to build and productionize machine learning pipelines.</span><br>
</div>
<div>
<span>-Fluent in SQL and Python; experience using Spark (pyspark) and working with both relational and non-relational databases.</span><br>
</div>
<div>
<span>-Demonstrated industry success in building and deploying machine learning pipelines, as well as feature engineering from semi-structured data.</span><br>
</div>
<div>
<span>-Solid understanding of the foundational concepts of machine learning and artificial intelligence.</span><br>
</div>
<div>
<span>-A desire to grow as an engineer through collaboration with a diverse team, code reviews, and learning new languages/technologies.</span><br>
</div>
<div>
<span>-2+ years of experience using version control, especially Git.</span><br>
</div>
<div>
<span>-Familiarity with Linux, AWS, Redshift.</span><br>
</div>
<div>
<span>-Deep learning experience preferred.</span><br>
</div>
<div><span>-Work experience with REST APIs, deploying microservices, and Docker is a plus.</span></div>
<div><br></div>
<div>
<span>What you can expect:</span><br>
</div>
<div><span><br></span></div>
<div>
<span>-Employ appropriate methods to develop performant machine learning models at scale, owning them from inception to business impact.</span><br>
</div>
<div>
<span>-Plan, engineer, and deploy both batch-processed and real-time data science solutions to increase user engagement with Doximity&rsquo;s products.</span><br>
</div>
<div>
<span>-Collaborate cross-functionally with data engineers and software engineers to architect and implement infrastructure in support of Doximity&rsquo;s data science platform.</span><br>
</div>
<div>
<span>-Improve the accuracy, runtime, scalability and reliability of machine intelligence systems</span><br>
</div>
<div><span>-Think creatively and outside of the box. The ability to formulate, implement, and test your ideas quickly is crucial.</span></div>
<div><br></div>
<div><span>Technical Stack:</span></div>
<div><span><br></span></div>
<div>
<span>-We historically favor Python and MySQL (SQLAlchemy), but leverage other tools when appropriate for the job at hand.</span><br>
</div>
<div>
<span>-Machine learning (linear/logistic regression, ensemble models, boosted models, deep learning models, clustering, NLP, text categorization, user modeling, collaborative filtering, topic modeling, etc) via industry-standard packages (sklearn, Keras, NLTK, Spark ML/MLlib, GraphX/GraphFrames, NetworkX, gensim).</span><br>
</div>
<div>
<span>-A dedicated cluster is maintained to run Apache Spark for computationally intensive tasks.</span><br>
</div>
<div>
<span>-Storage solutions: Percona, Redshift, S3, HDFS, Hive, Neo4j, and Elasticsearch.</span><br>
</div>
<div>
<span>-Computational resources: EC2, Spark.</span><br>
</div>
<div><span>-Workflow management: Airflow.</span></div>
<div><br></div>
<div>
<span>Fun facts about the Data Science team:</span><br>
</div>
<div><span><br></span></div>
<div>
<span>-We have one of the richest healthcare datasets in the world.</span><br>
</div>
<div>
<span>-We build code that addresses user needs, solves business problems, and streamlines internal processes.</span><br>
</div>
<div>
<span>-The members of our team bring a diverse set of technical and cultural backgrounds.</span><br>
</div>
<div>
<span>-Business decisions at Doximity are driven by our data, analyses, and insights.</span><br>
</div>
<div>
<span>-Hundreds of thousands of healthcare professionals will utilize the products you build.</span><br>
</div>
<div>
<span>-A couple times a year we run a co-op where you can pick a few people you'd like to work with and drive a specific company goal.</span><br>
</div>
<div><span>-We like to have fun - company outings, team lunches, and happy hours!</span></div>
<p><strong>To apply:</strong> <a href="https://grnh.se/y8yls8hh1">https://grnh.se/y8yls8hh1</a></p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
March 29, 2018 at 04:15PM
